app/models/project_services/hipchat_service.rb

```diff
@@ -34,6 +34,8 @@ def fields
     [
       { type: 'text', name: 'token',     placeholder: 'Room token' },
       { type: 'text', name: 'room',      placeholder: 'Room name or ID' },
+      { type: 'text', name: 'api_version',
+        placeholder: "v1 for group-level api token, v2 for account-level api token. default is v2" },
```

Comment:
Line is too long. [101/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.